### PR TITLE
<List/> and <Layout/> Styling

### LINKED ISSUE #34: Style <List /> and <Layout /> components
Make the /list page shine! :) Pending design decisions that we make as a team.

### PR DESCRIPTION
**Pull request by michellerenehey**
## Description
This was a big styling haul!!! This PR includes a refined style of the homepage, an overhaul of the layout component (which includes an update of the headers and footers), as well as styling of the list component (both when the list is empty and when there are items in it). 

**NOTE. This PR does not include styling the `ListItem` component!**

## Related Issue
Closes #34. The project ended up being much bigger than expected, so I broke out the styling of the `<ListItem>` component into another ticket. 

## Acceptance Criteria
- move the SHOP-ADE image to the homepage only
- create a new header and put it in the `layout` component: this header includes list name, a spot for the `copy list name` button, and log out functionality
- the layout component now conditionally renders either just the homepage (with the SHOP-ADE image), or this new header with the rest of the app.
- remove logout functionality and list names from `list` component (into header) 
- style the footer
- on `list` component, fully style the page when there is an empty list
- on `list` component, fully style the page when there IS a list
- include a scroll bar when the list is too long to fit into container 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|    | :sparkles: New feature     |
|   ✓  | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |
|  ✓   | 💅 : Styling              |

### Before
Homepage before

Empty list before

List with items before

### After
Homepage after updates

Empty list after

List with items after updates

## Testing Steps / QA Criteria

1. `git checkout main` then `git pull`
2. `git checkout MN-MK-list-styling`
3. `npm run start`
3. navigate around the app

### CODE REVIEW

File: src/views/Home.css [hunk 1]
```diff
@@ -1,16 +1,35 @@
...
 	background-color: var(--blue);
-	height: 80vh;
 	display: flex;
 	flex-direction: column;
 	justify-content: space-evenly;
 	align-items: center;
 }
 
+.home-header {
+	position: sticky;
```
Comment (mira-kine) [inline src/views/Home.css:10]: styling goddess using position sticky

**Review by mira-kine — APPROVED** [review]
I love your use of responsive design, thank you for cleaning up the code and creating readable, understandable containers!

**Comment by N-Zubko** [comment]
<Screen shot demonstrating weird small square in bottom right part of the app" src="https://user-images.githubusercontent.com/63440229/187056575-971daaa8-82c8-4ce3-afee-48e9534c167f.png">

There's a small square in the bottom right part of the screen under Inactive

**Comment by nickytonline** [comment]
> There's a small square in the bottom right part of the screen under Inactive
> 
> 
> 
> Screen shot demonstrating weird small square in bottom right part of the app

It appears to be a checkbox outside of the mapping of the other checkboxes in list items.

**Review by N-Zubko — APPROVED** [review]

**Comment by michellerenehey** [comment]
> > There's a small square in the bottom right part of the screen under Inactive
> > Screen shot demonstrating weird small square in bottom right part of the app
> 
> It appears to be a checkbox outside of the mapping of the other checkboxes in list items.

@N-Zubko @nickytonline 
Oh thank you for noticing that! It was driving me nuts. After some digging, I've found that it's actually the base/bottom of the scrollbar I customized. I just added a `::-webkit-scrollbar-corner {
	background-color: var(--blue);
}` to blend in and it works great! 

Scrollbar fix

Nick I think that extra input label is fixed in Mira's branch!

**Review by Trevor-Rezac — APPROVED** [review]
Everything is looking great and I love the direction the theme took! Approving this PR.

**Review by lindseyindev — APPROVED** [review]
I really like the style choices you made, I think the containers that are dotted like pacman food are very clever! 
There is a lot going on and I went through the dev inspector primarily for my testing and I didn't see any unused CSS properties, great work here!

**Pull request merged**